### PR TITLE
0080-feel-getvalue-function

### PR DESCRIPTION
**Pull request by StrayAlien**

### CODE REVIEW

File: TestCases/compliance-level-3/0080-feel-getvalue-function/0080-feel-getvalue-function-test-01.xml [hunk 1]
```diff
@@ -0,0 +1,123 @@
...
+            <expected>
+                <value xsi:nil="true"></value>
+            </expected>
+        </resultNode>
+    </testCase>
+-->
+
+    <testCase id="decision_008">
+        <description>badly named params</description>
+        <resultNode errorResult="true" name="decision_008" type="decision">
```
Comment (tarilabs) [inline TestCases/compliance-level-3/0080-feel-getvalue-function/0080-feel-getvalue-function-test-01.xml]: Decision node `decision_008` is commented out in DMN model. I believe this test ought to be commented out. In fact it seems the copy-paste from line 74.

Comment (StrayAlien) [reply TestCases/compliance-level-3/0080-feel-getvalue-function/0080-feel-getvalue-function-test-01.xml]: Thanks @tarilabs. Amended.

**Comment by agilepro** [comment]
Not yet voted on.  Still waiting.

**Comment by agilepro** [comment]
Balloted.  If voted up, can argue that this issue has been clarified and then we can check those in.  

Merge it when approved.

File: TestCases/compliance-level-3/0080-feel-getvalue-function/0080-feel-getvalue-function.dmn [hunk 2]
```diff
@@ -0,0 +1,130 @@
...
+        </literalExpression>
+    </decision>
+
+    <decision name="decision_008" id="_decision_008">
+        <description>Tests FEEL expression: 'get value(k:"a", m:{a: "foo"})' and expects result: 'null'</description>
+        <question>Result of FEEL expression 'get value(k:"a", m:{a: "foo"})'?</question>
+        <allowedAnswers>null</allowedAnswers>
+        <variable typeRef="string" name="decision_008"/>
+        <literalExpression>
+            <text>get value(k:"a", m:{a: "foo"})</text>
```
Comment (tarilabs) [inline TestCases/compliance-level-3/0080-feel-getvalue-function/0080-feel-getvalue-function.dmn]: the parameter names in this test looks wrong, and inconsistent with the previous test. However both are commented out. Overall test file approved from my side.

**Comment by opatrascoiu** [comment]
looks good to me

**Comment by tarilabs** [comment]
Based on https://github.com/dmn-tck/tck/pull/236#issuecomment-480343235 https://github.com/dmn-tck/tck/pull/236#pullrequestreview-229942394 https://github.com/dmn-tck/tck/pull/236#issuecomment-486732479 will merge.

**Pull request merged**